docs/source/outline.rst

```diff
@@ -0,0 +1,72 @@
+.. _project_outline:
+
+===============
+Project Outline
+===============
+
+Current plan for the StreamWebs rewrite. For a visual representation, 
+refer to `this`_.
+
+On the site's homepage, the user will be able to navigate to one of the
+following pages:
+
+* Login
+* Analyze/Learn
+* Resources
+
+.. _this: http://i.imgur.com/XqOmLQn.jpg
+
+
+Login
+-----
+
+After login, the user will be taken to a central ``Sites`` page.
+
+.. note::
+
+  Alongside **Users**, basic CRUD will need to be implemented for **Resources**
+  and **Map/Sites** as well.
+
+Analyze/Learn
+-------------
+
+The Analyze/Learn page will also redirect the user to ``Sites``.
```

Comment:
Does `Analyze/Learn` have anything else, or is it just another name for the `Sites` page?